### ISSUE TITLE
When using typing input blocks aren't added to the (un|re)drop list

### ISSUE THREAD

**Issue by cycomachead** [jmoenig/Snap--Build-Your-Own-Blocks#1569]
If you're in typing mode and add blocks they aren't added as undroppable actions. It seems to be like hitting `return` should be the equivalent to dropping a block that would otherwise be undroppable.

**Comment by jmoenig** [comment]
Yep, that's a loose end. The only thing I did add was that when you hit backspace to delete a block in keyboard entry mode you can undelete it with ctrl-z
